jerry-core/lit/lit-unicode-ranges.inc.h

```diff
@@ -2436,12 +2436,16 @@ LIT_UNICODE_RANGE_ZS (0x00A0, 0x00A0) /* NO-BREAK SPACE */
 
 LIT_UNICODE_RANGE_ZS (0x1680, 0x1680) /* OGHAM SPACE MARK */
 
+LIT_UNICODE_RANGE_ZS (0x180E, 0x180E) /* MONGOLIAN VOWEL SEPARATOR */
+
 LIT_UNICODE_RANGE_ZS (0x2000, 0x200B) /*     EN QUAD
                                               <--->
                                          ZERO WIDTH SPACE */
 
 LIT_UNICODE_RANGE_ZS (0x202F, 0x202F) /* NARROW NO-BREAK SPACE */
 
+LIT_UNICODE_RANGE_ZS (0x205F, 0x205F) /* MEDIUM MATHEMATICAL SPACE */
```

Comment:
The comment in the file header says that `The range lists were generated using tools/print-unicode-ranges.sh script from UnicodeData-3.0.0.txt`, but this change makes the statement incorrect. Could you, please, update the header, and also, add special comment for manually added ranges, describing the ranges origin? Maybe, the manually added ranges should also be grouped at start of range block (near `ifdef`).